persistence-cassandra/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/cassandra/CassandraPersistenceComponents.scala

```diff
@@ -21,6 +26,19 @@ trait WriteSideCassandraPersistenceComponents extends WriteSidePersistenceCompon
   override lazy val persistentEntityRegistry: PersistentEntityRegistry =
     new CassandraPersistentEntityRegistry(actorSystem)
 
+  def serviceLocator: Option[ServiceLocator]
```

Comment:
@jroper This is the part I'm uncertain about. The `ServiceLocator` is optional, see the dance in the above javadsl.CassandraPersistenceModule. I'm not sure when the `ServiceLocator` is available or not, but at least for some testing it is good to be able to run without `ServiceLocator`.

If it should be an optional dependency as we have in the javadsl I'm not sure how MacWire would handle it.